assets/app/views/_project-page.html

```diff
@@ -16,10 +16,8 @@
     <div class="container-fluid">
       <div class="sidebar-header sidebar-header-bleed-left sidebar-header-bleed-right">
         <div class="actions pull-right">
-          <!-- If we have any header actions for the sidebar in the future, they would go here
-              and be links like below
-              <a href="#">Example</a>
-          -->
+	  <a href="#" ng-click="overviewToggle()" ng-if="overviewMode == 'tiles'">Topology</a>
```

Comment:
Hmm maybe:
http://fortawesome.github.io/Font-Awesome/icon/list
or
http://fortawesome.github.io/Font-Awesome/icon/th-list  leaning towards
this one

It would be nice if we had a graph icon that was more representative,
something like

https://www.google.com/search?q=graph+icon&espv=2&biw=1920&bih=1017&tbm=isch&tbo=u&source=univ&sa=X&ved=0CB0QsARqFQoTCI_9pYaVpscCFYyggAodFB8KKQ#tbm=isch&q=directed+graph+icon&imgrc=HIeoKVqIpM9L6M%3A

maybe we can get something added to patternfly

On Thu, Aug 13, 2015 at 2:11 AM, Stef Walter <notifications@github.com>
wrote:

> In assets/app/views/_project-page.html
> <https://github.com/openshift/origin/pull/2843#discussion_r36943427>:
>
> > @@ -16,10 +16,8 @@
> >      <div class="container-fluid">
> >        <div class="sidebar-header sidebar-header-bleed-left sidebar-header-bleed-right">
> >          <div class="actions pull-right">
> > -          <!-- If we have any header actions for the sidebar in the future, they would go here
> > -              and be links like below
> > -              <a href="#">Example</a>
> > -          -->
> > +	  <a href="#" ng-click="overviewToggle()" ng-if="overviewMode == 'tiles'">Topology</a>
>
> So I guess use a button group?
>
> https://www.patternfly.org/widgets/#button-groups
>
> Which icons should I use? fa-table and fa-sitemap? Hmmm, although Cockpit
> uses fa-sitemap for the topology view, Openshift uses it for 'Details'. Any
> ideas?
>
> http://fortawesome.github.io/Font-Awesome/icon/table/
> http://fortawesome.github.io/Font-Awesome/icon/sitemap/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/2843/files#r36943427>.
>